x-pack/legacy/plugins/reporting/server/browsers/chromium/driver_factory/index.ts

```diff
@@ -101,7 +100,7 @@ export class HeadlessChromiumDriverFactory {
         viewport,
         verboseLogging: this.logger.isVerbose,
         disableSandbox: this.browserConfig.disableSandbox,
-        proxyConfig: this.browserConfig.proxy,
```

Comment:
looks like this was a bug - it should be `proxy` per https://www.elastic.co/guide/en/kibana/current/reporting-settings-kb.html#reporting-chromium-settings